workflow/training/whisper.py

```
@@ -35,174 +29,133 @@ def __init__(self, model_name: str, version: str, args):
             train_dataset = dataset["train"]
-            print("Dataset splits:", dataset.keys())
-            for split_name, split_data in dataset.items():
-                print(f"\nSplit: {split_name}")
-                print("Columns:", split_data.column_names)
-                print("Number of rows:", len(split_data))
-                
+        
         if "audio" not in train_dataset.column_names:
             audio_column = train_dataset.column_names[0]
             train_dataset = train_dataset.rename_column(audio_column, "audio")
+            logger.info(f"Renamed column to 'audio'")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)